limhoff-r7/limhoff-r7.github.io
# Reading legacy code to harmonize Ruby and Rails code loaders

MSP-12043

Draft of "Reading legacy code to harmonize Ruby and Rails code loaders".  It will be published after [Austin.rb on 2015-02-02](http://www.meetup.com/austinrb/events/219854095/).  This article will serve as the basis for the slide presentation given at Austin.rb.
# Verification Steps

**NOTE: you can see the rendered Markdown inline using [this link](https://github.com/limhoff-r7/limhoff-r7.github.io/pull/1/files?short_path=80d0ef0#diff-2), but it's still best to use jekyll as it will generate the Table of Contents and process the YAML front matter**
- [x] `mkdir -p ~/git/limhoff-r7`
- [x] `cd ~/git/limhoff-r7`
- [x] `git clone git@github.com:limhoff-r7/limhoff-r7.github.io`
- [x] `cd limhoff-r7.github.io`
- [x] `rvm use ruby-2.1@limhoff-r7.github.io --create`
- [x] `git fetch origin`
- [x] `git checkout -b chore/MSP-12043/reading-legacy-code-to-harmonize-ruby-and-rails-code-loaders origin/chore/MSP-12043/reading-legacy-code-to-harmonize-ruby-and-rails-code-loaders`
- [x] `git merge master`
- [x] `jekyll serve --drafts`
- [x] `open http://localhost:4000`
- [x] Click "Reading legacy code to harmonize Ruby and Rails code loaders"
- [x] VERIFY no spelling errors
- [x] VERIFY no grammar errors
- [x] VERIFY all images display
- [x] VERIFY all links work
- [x] VERIFY youtube video displays in page and remotely
- [x] VERIFY everything is explained enough (this talk needs to be newbie friendly)
- [x] VERIFY nothing is over explained (this talk needs to not be overly boring)
